README.md

```diff
@@ -34,10 +34,15 @@ Mouse navigation:
 
 Arch Linux:
 
+For users who are using KF6/Plasma 6
 ```sh
 yay -S kwin-effects-cube-git
 ```
-
+For users who are using Plasma 5
+```sh
+yay -S kwin-effects-cube
+```
+Hint: Please reload kwin after installation to make this effect work.
```

Comment:
This should be unnecessary

Reply:
But on my Plasma 5, it's a must step. We should relaod kwin using `kwin_x11 --replace` or `kwin_wayland --replace` to activate this effect, refer https://github.com/zzag/kwin-effects-cube/issues/6#issuecomment-1328268440

Reply:
That doesn't make sense. Meta+C should work after the effect is enabled in system settings. If not, perhaps there's a shortcut conflict

Reply:
Please wait some time, I'll ask some friends to test both on Xorg and Wayland.

Reply:
There could be something wrong on my Plasma config. My friends announce that it works without kwin reload.